<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddQuesDialog</class>
 <widget class="QDialog" name="AddQuesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>95</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>95</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>95</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#AddQuesDialog{
	background: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel{
	border: 2px solid rgb(26, 222, 209);
	color: #FFFFFF;
	background-color: rgb(26, 222, 209);
}</string>
       </property>
       <property name="text">
        <string>题目类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbbTypes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 25 10pt '微软雅黑';
	color: #FFFFFF;
	border: 2px solid rgb(24, 204, 192);
	background-color: rgb(20,196,188);
}
QPushButton:hover{
	background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>AddQuesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
